app/src/assets/css/Selectable.css

```diff
@@ -10,6 +10,9 @@
   flex-direction: column;
   border-radius: 2px;
   transition: 0.25s border-color;
+  user-select: none;
+  cursor: pointer;
+  cursor: hand;
```

Comment:
What about when the components doesnt look like regular buttons? 

These components look very much like ours, and they have pointers. http://materializecss.com/collapsible.html